### PR TITLE
run e2e tests for beta-release unless skipped

### PR DESCRIPTION
**Pull request by grahamlangford**
## What does this PR do?

- Part of #8343

## Reviewer Tips

- Changes in ci.yaml and release.yaml were just cleanup

## Checklist

- [x] Designate a primary reviewer @fungairino

### CODE REVIEW

**Review by fungairino — APPROVED** [review]

**Comment by mnholtz** [comment]
I made some changes to env variables recently that this branch will probably need to pull in. Merging with main to look into fixing that

File: .github/workflows/beta-release.yaml [hunk 1]
```diff
@@ -65,9 +69,58 @@ jobs:
...
 
+  end-to-end-tests:
+    name: end-to-end-tests
+    if: ${{ github.event.inputs.skip_tests != 'true' }}
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    env:
+      SHADOW_DOM: open
+      SERVICE_URL: https://app.pixiebrix.com
+      REQUIRE_OPTIONAL_PERMISSIONS_IN_MANIFEST: 1
```
Comment (mnholtz) [inline .github/workflows/beta-release.yaml:80]: Now that I'm looking at this, where is the test username/password?

**Review by mnholtz — CHANGES_REQUESTED** [review]
Sorry, maybe I'm misunderstanding something. But I don't see `E2E_TEST_USER_EMAIL_UNAFFILIATED` or `E2E_TEST_USER_PASSWORD_UNAFFILIATED`.

Also, I want to make sure that we update the required env variables per changes made in https://github.com/pixiebrix/pixiebrix-extension/pull/8341

In particular, I think we'll need to add ` DEV_EVENT_TELEMETRY` and `DATADOG_CLIENT_TOKEN`

**Comment by twschiller** [comment]
@grahamlangford see @mnholtz's change request comments

**Comment by grahamlangford** [comment]
> Sorry, maybe I'm misunderstanding something. But I don't see `E2E_TEST_USER_EMAIL_UNAFFILIATED` or `E2E_TEST_USER_PASSWORD_UNAFFILIATED`.
> 
> Also, I want to make sure that we update the required env variables per changes made in #8341
> 
> In particular, I think we'll need to add ` DEV_EVENT_TELEMETRY` and `DATADOG_CLIENT_TOKEN`

Done. `DATADOG_CLIENT_TOKEN` was already included

**Review by mnholtz — APPROVED** [review]

**Pull request merged**